app/apphandlers/processorAppHandler.go

```diff
@@ -339,18 +345,21 @@ func (processor *ProcessorApp) LegacyStart(ctx context.Context, options *app.Opt
 	if processor.App.Features().Migrator != nil {
 		if migrationMode == db.IMPORT || migrationMode == db.EXPORT || migrationMode == db.IMPORT_EXPORT {
 			startProcessorFunc := func() {
-				g.Go(func() error {
-					clearDB := false
-					StartProcessor(ctx, &clearDB, enableProcessor, &gatewayDB, &routerDB, &batchRouterDB, &procErrorDB, reportingI, multitenantStats)
-
-					return nil
-				})
+				clearDB := false
```

Comment:
Any idea why we would prefer passing a reference to a bool value instead of the value itself?

Reply:
Nope, in my opinion it is small enough to be passed across the stack instead of adding pressure to the GC :man_shrugging: 

More stuff to change for next time :broom: :stuck_out_tongue_winking_eye: